libs/explorers/comparison-tool/src/lib/comparison-tool-table/base-table/base-table.component.ts

```diff
@@ -39,6 +39,9 @@ export class BaseTableComponent {
   shouldPaginate = input<boolean>(true);
   shouldShowNoDataMessage = input<boolean>(true);
   columnWidth = input<string>('auto');
+  totalRecords = input<number | undefined>(undefined);
```

Comment:
We already inject the CT service in this component -- can we get this value directly from the CT service instead of using prop drilling? 


```suggestion
  totalRecords = this.comparisonToolService.totalResultsCount;
```